pymeasure/instruments/rigol/rigol_dho804.py

```
@@ -0,0 +1,511 @@
+        ":CHAN{ch}:BWL %s",
+        """Control the bandwidth limit: ``"OFF"``, ``"20M"``, or ``"100M"``.""",
+        validator=strict_discrete_set,
+        values=["OFF", "20M", "100M"],
+    )
+
+    scale = Channel.control(
+        ":CHAN{ch}:SCAL?",
+        ":CHAN{ch}:SCAL %g",
+        """Control the vertical scale in V/div (float).
+
+        Valid range: 500 µV/div – 10 V/div for a 1x probe.
```

Ruff (0.15.12)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)